locale_file_health_check.py

```
@@ -0,0 +1,65 @@
+class LocaleFileHealthCheck(HealthCheck):
```

Flake8 (7.3.0)

[error] undefined name 'HealthCheck'

(F821)